### PR TITLE
CB-10355: Add Python installation instructions to Readme.md file

### PR DESCRIPTION
**Pull request by sarangan12**
Modified the Readme.md file to include instructions for Python
installation. This is done because Python is required for a successful
build of the cordova-docs project

### CODE REVIEW

**Comment by sarangan12** [comment]
@dblotsky @riknoll Can you please review these changes and accept the pull request?

**Comment by riknoll** [comment]
LGTM but I'll default to @dblotsky on this since he knows more about the docs dev environment than I do.

File: README.md [hunk 1]
```diff
@@ -44,6 +44,37 @@ Run the commands from [this site][ruby_linux] that apply to your Linux distribut
 Verify the installation by running:
 
     ruby --version
+## Python
```
Comment (dblotsky) [inline README.md]: Nitpick: Please add a space above this line.

**Comment by dblotsky** [comment]
Aside from the above comment,  LGTM.

**Comment by sarangan12** [comment]
@dblotsky Can you check and merge when you have time?

**Comment by nikhilkh** [comment]
I just setup my machine and these instructions are accurate for windows

**Pull request closed without merging**